psst-gui/src/ui/mod.rs

```diff
@@ -42,7 +42,7 @@ pub mod utils;
 pub fn main_window(config: &Config) -> WindowDesc<AppState> {
     let win = WindowDesc::new(root_widget())
         .title(compute_main_window_title)
-        .with_min_size((theme::grid(65.0), theme::grid(25.0)))
+        .with_min_size((theme::grid(65.0), theme::grid(65.0)))
```

Comment:
I think a better minimum height would be `46.0`. It allows just enough room for one playlist to be displayed without anything else getting squished.

Reply:
If it was the default size I could see that, but it's the minimum, so I think it should be as small as possible while still being usable.

Personally, I only have one playlist, so it would end up with a bunch of extra space that isn't being used.

Reply:
Okay, I'm okay with this. In a future PR I think we ought to hide the psst icon after some height threshold to free up space.

Reply:
I don't think 46 makes a single playlist possible to view... I think 50 would work considering double-lined playlists as well.